src/containers/create-new/index.tsx

```diff
@@ -7,7 +7,7 @@ import Image from 'next/image';
 import ProductModal from '@components/modals/product-modal';
 import PropTypes from 'prop-types';
 import React, { useState } from 'react';
-import SpinnerContainer from '@ui/spinner-container';
+import { SpinnerContainer } from '@ui/spinner-container/spinner-container';
```

Comment:
Like here for instance, what is the benefit of changing something that followed a rule?

Reply:
Here, I did a change in the name file from index.tsx to spinner-container.tsx